geoextent/lib/handleVector.py

```diff
@@ -47,7 +52,47 @@ def getTemporalExtent(filepath):
     input "path": type string, file path to vector file
     '''
 
-    return None
+    global parsed_time
+    dataset = ogr.Open(filepath)
+    layer = dataset.GetLayer()
+    layerDefinition = layer.GetLayerDefn()
+
+    field_names = []
+    for i in range(layerDefinition.GetFieldCount()):
+        field_names.append(layerDefinition.GetFieldDefn(i).GetName())
+
+    match_list = []
+    for x in search["time"]:
+        for j in field_names:
+            term = re.compile(x, re.IGNORECASE)
+            match = term.search(j)
+            if match is not None:
+                match_list.append(j)
+
+    if len(match_list) == 0:
+        logger.debug('The file {} has no TemporalExtent'.format(filepath))
+        return None
+    else:
+        timelist = []
+        for time_feature in match_list:
+            for feat in layer:
+                time = feat.GetField(time_feature)
+                if time is not None:
+                    timelist.append(time)
+
+    if len(timelist) == 0:
+        logger.debug('The file {} has no TemporalExtent'.format(filepath))
+        return None
+    else:
+        parsed_time = hf.date_parser(timelist)
+
+    if parsed_time is None:
+        logger.debug('The file {} has no recognizable TemporalExtent'.format(filepath))
+        return None
+    else:
+        tbox = [min(parsed_time).strftime('%Y-%m-%d'), max(parsed_time).strftime('%Y-%m-%d')]
```

Comment:
Please make the time format a global variable defined somewhere else, so we only need to change it in one place, and we're not forgetting about the multiple places when implementing #80